controllers/controllers.js

```diff
@@ -16,11 +17,18 @@ exports.getArticleById = (req, res, next) => {
 };
 
 exports.getUsers = (req, res, next) => {
-  selectUsers().then((topics) => {
-    res.status(200).send(topics)
+  selectUsers().then((users) => {
+    res.status(200).send(users)
```

Comment:
Please send this back on a key of users. At the moment, it will respond with just an array. We want an object with a key of users, and a value of the array of users, please.